contrib/mesos/pkg/scheduler/podtask/pod_task.go

```diff
@@ -26,15 +26,20 @@ import (
 	annotation "github.com/GoogleCloudPlatform/kubernetes/contrib/mesos/pkg/scheduler/meta"
 	"github.com/GoogleCloudPlatform/kubernetes/contrib/mesos/pkg/scheduler/metrics"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/resourcequota"
 	"github.com/gogo/protobuf/proto"
+
 	log "github.com/golang/glog"
 	mesos "github.com/mesos/mesos-go/mesosproto"
 	mutil "github.com/mesos/mesos-go/mesosutil"
 )
 
+type MegaBytes float64
+type CPUShares float64
+
 const (
-	DefaultContainerCpus = 0.25 // initial CPU allocated for executor
-	DefaultContainerMem  = 64   // initial MB of memory allocated for executor
+	DefaultContainerCpus = CPUShares(0.25) // initial CPU allocated for executor
```

Comment:
since these are really tiny numbers for pods whose resources are supposed to be unbounded, do you think it's worth logging a warning? or info? to let the user know about the minuscule resources we're giving to them?